### ISSUE TITLE
 Cannot find module!!!!!!!

### ISSUE THREAD

**Issue by sajjadmrx** [Bannerets/tdl#98]
(node:4944) UnhandledPromiseRejectionWarning: Error: Cannot find module '../build/Release/td.node'

**Comment by Bannerets** [comment]
Don't yell... Were there any errors when you executed npm install?

**Comment by BAE86AEC1946B6AB60CCD7452DC811F0** [comment]
me too.    no errors  ` npm install`

**Comment by BAE86AEC1946B6AB60CCD7452DC811F0** [comment]
```
npm install
npm WARN tg@1.0.0 No description
npm WARN tg@1.0.0 No repository field.

audited 50 packages in 0.741s

1 package is looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```


```
Error: Cannot find module '../build/Release/td.node'
Require stack:
- ***\node_modules\tdl-tdlib-addon\dist\index.js
- ***\node_modules\tdl-tdlib-addon\index.js
- ***\index.js
```

**Comment by Bannerets** [comment]
Looks like new npm v7 doesn't output node-gyp's log (i.e. log of the postinstall script) if the process exited with zero exit code. I fixed the issue in tdl-tdlib-addon v1.0.1, install it again. You'll get the error log. Most likely Python or a C++ compiler is not installed on your device.
